codalab/worker/file_util.py

```diff
@@ -336,6 +469,15 @@ def get_path_size(path, exclude_names=[], ignore_nonexistent_path=False):
     If ignore_nonexistent_path is True and the input path is nonexistent, the value
     0 is returned. Else, an exception is raised (FileNotFoundError).
     """
+    if parse_linked_bundle_url(path).uses_beam:
+        # On Azure, use Apache Beam methods, not native os methods,
+        # to get the path size.
+
+        # Get the size of the specified path (file / directory).
+        # This will only get the right size of files, not of directories (but we don't
+        # store any bundles as directories on Azure).
+        return get_file_size(path)
```

Comment:
Would it make sense to reuse the `get_path_size` function in `codalab.worker.file_util`?

Reply:
No, this code is within the `get_path_size` function in `codalab.worker.file_util`, so we can't do that.